src/hal/i_components/latencybins.icomp

```diff
@@ -25,6 +25,10 @@ Usage:
    The latency pin outputs the instantaneous latency.
 
 Maintainers note: hardcoded for MAXBINNUMBER==1000
+
+NB.  Since there is no hal_64_t yet, this component has been left 'as is'
+using __64 type etc
```

Comment:
I think __32 could be exchanged by hal_s32_t. The two __64 variables are used to store the timestamps which is fine. The __32 variables is used for output only.